_layouts/default.html

```diff
@@ -46,6 +46,7 @@
             <script src="/media/system/js/html5fallback.js" type="text/javascript"></script>
             <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.min.js"></script>
             <script type="text/javascript" src="/plugins/system/t3/base-bs3/js/respond.min.js"></script>
+            <script type="text/javascript" src="/scripts/svgxuse.js"></script>
```

Comment:
Uups, just seen this. Ok then, but still which browsers does this polyfill supports? :)

Reply:
from https://github.com/Keyamoon/svgxuse

"._.. IE (9, 10, 11) fails to load symbol-defs.svg in the example above. svgxuse goes through every <use> element. If the element is referencing an external SVG and the browser has failed to load it, the script sends a GET request to grab and prepend the SVG to <body>..._"

by nature polyfills are api's that instert a functionality that a browser natively does not support, so it should work fine. 

remember that font-awesome is also a solution that [supports a range of browsers](https://fontawesome.com/how-to-use/on-the-web/other-topics/browser-support) , we are simply reducing the risk of incompatibility by using a more modern approach with a fallback option.